src/RpcClient.ts

```diff
@@ -506,17 +506,21 @@ export class RpcClient {
   /**
    * Build and send an optimized transaction, and handle its confirmation status
    * @param {TransactionInstruction[]} instructions - The transaction instructions
-   * @param {Keypair} fromKeypair - The sender's keypair
+   * @param {Signer[]} signers - The transaction signers. the first signer should be the fee payer
```

Comment:
Nit: Can you rephrase this to `The transaction's signers. The first signer should be the fee payer`